examples/org-website/custom_serializer_sample.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"reflect"
+	"time"
+
+	"github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/serialization"
+)
+
+type Employee struct {
+	Surname string
+}
+
+type EmployeeCustomSerializer struct{}
+
+func (e EmployeeCustomSerializer) ID() (id int32) {
+	return 45392
+}
+
+func (e EmployeeCustomSerializer) Read(input serialization.DataInput) interface{} {
+	surname := input.ReadString()
+	return &Employee{Surname: surname}
+}
+
+func (e EmployeeCustomSerializer) Write(output serialization.DataOutput, object interface{}) {
+	employee, ok := object.(*Employee)
+	if !ok {
+		panic("can serialize only Employee")
+	}
+	output.WriteString(employee.Surname)
+}
+
+func main() {
+	// Configure serializer
+	config := hazelcast.NewConfig()
```

Comment:
Just out of curiousity, why do we have `NewConfig()` if we recommend initializing the struct directly?

Reply:
I've left `NewConfig` in v1 since in the future we may have similar specialized config creation functions, like `NewCloudConfig("my-cluster", "my-token")`.